tests/integration/test_stage_and_submit.py

```diff
@@ -0,0 +1,532 @@
+        )
+
+        if status==200:
+            response_target_values = {
+                key: val
+                for key, val in response.json.items()
+                if key in request_body
+            }
+
+            assert response_target_values == request_body, (
+                "Requested changes did not take place."
+            )
+            
+            return response.json
+
+
+    def post_presubvalidation(
+        self,
+        auth:AuthFixture,
+        metadataset_ids: List[str],
+        file_ids: List[str],
+        label:str="test_submission",
+        status:int=204
+    ):
+        # request params:
+        request_body = {
+            "metadatasetIds": metadataset_ids,
+            "fileIds": file_ids,
+            "label": label
+        }
+
+        response = self.testapp.post_json(
+            base_url + f"/presubvalidation",
+            headers=auth.header,
+            params=request_body,
+            status=status
+        )
+
+
+    def post_submission(
+        self,
+        auth:AuthFixture,
+        metadataset_ids: List[str],
+        file_ids: List[str],
+        label:str="test_submission",
+        status:int=200
+    ):
+        # request params:
+        request_body = {
+            "metadatasetIds": metadataset_ids,
+            "fileIds": file_ids,
+            "label": label
+        }
+
+        response = self.testapp.post_json(
+            base_url + f"/submissions",
+            headers=auth.header,
+            params=request_body,
+            status=status
+        )
+
+        if status==200:
+            return response.json
+
+
+    def get_all_submissions(
+        self,
+        auth:AuthFixture,
+        group_id:str,
+        expected_submission_uuid:Optional[str]=None,
+        status:int=200
+    ):
+        # with open(file_path, 'rb') as file_to_upload:
+        response = self.testapp.get(
+            base_url + f"/groups/{group_id}/submissions",
+            headers=auth.header,
+            status=status
+        )
+
+        if status == 200:
+            if expected_submission_uuid:
+                submission_uuids = [
+                    sub["id"]["uuid"]
+                    for sub in response.json
+                ]
+                assert expected_submission_uuid in submission_uuids, (
+                    "Expected submission was not contained in response."
+                )
+            return response.json
+
+
+    def test_main_submission_senario(self):
+        """Tests the standard usage senario for staging and 
+        submitting files and metadata."""
+        user = self.users["user_a"]
+        
+        # post metadataset:
+        metadataset_ids = [
+            self.post_metadata(
+                auth=user.auth, 
+                metadata_record=rec
+            )["id"]["site"]
+            for rec in self.metadata_records
+        ]
+        
+        # get metadatasets and compare to original records:
+        _ = [
```

Comment:
is there a specific reason, why this is assigned to `_` instead of just putting the listcomp (or function calls below) without assignment?